js/lib/components/feedback/tooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faQuestion } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { VirtualElement } from "@popperjs/core";
+import React, { MouseEventHandler, useState } from "react";
+import { usePopper } from "react-popper";
+
+import "../../css/feedback.scss";
+
+// load question mark SVG icon
+library.add(faQuestion);
+
+type SelectionTooltipProps = {
+  reference: VirtualElement;
+  toggleModal?: () => void;
+  text?: string;
+};
+const SelectionTooltip: React.FC<SelectionTooltipProps> = ({ reference, toggleModal, text }) => {
+  const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);
+  const [arrowElement, setArrowElement] = useState<HTMLElement | null>(null);
+  const { styles, attributes } = usePopper(reference, popperElement, {
+    placement: "top",
+    modifiers: [
+      { name: "offset", options: { offset: [0, 8] } },
+      { name: "arrow", options: { element: arrowElement } },
+    ],
+  });
+
+  const handleTooltipClick: MouseEventHandler<HTMLDivElement> = ev => {
+    // prevent loss of selected text when user clicks on tooltip
+    ev.preventDefault();
+  };
+
+  return (
+    <div
+      ref={setPopperElement}
+      className="pop"
+      onMouseDown={handleTooltipClick}
+      style={styles.popper}
+      {...attributes.popper}
+    >
+      {text ? (
+        <div>{text}</div>
+      ) : (
+        <div className="pop-button" onClick={toggleModal} title="Provide feedback on this content">
+          <FontAwesomeIcon icon="question" />
```

Comment:
Would a simple ❓ emoji suffice? Or does it not look as good as the FA icon?

Reply:
Oh yeah, emojis should work